function-sidecar/src/main/java/io/sk8s/sidecar/HttpDispatcher.java

```diff
@@ -16,19 +16,37 @@
 
 package io.sk8s.sidecar;
 
+import org.springframework.retry.backoff.ExponentialBackOffPolicy;
+import org.springframework.retry.policy.SimpleRetryPolicy;
+import org.springframework.retry.support.RetryTemplate;
 import org.springframework.web.client.RestTemplate;
 
+import javax.annotation.PostConstruct;
+
 /**
  * @author Mark Fisher
  */
 public class HttpDispatcher implements Dispatcher {
 
-	private static final String INVOKER_URL = "http://localhost:8080/";
+    private static final String INVOKER_URL = "http://localhost:8080/";
+
+    private final RestTemplate restTemplate = new RestTemplate();
+    private final RetryTemplate retryTemplate = new RetryTemplate();
+    private final ExponentialBackOffPolicy backOffPolicy = new ExponentialBackOffPolicy();
```

Comment:
What's the rationale for using exponential backoff here? If the purpose is to handle the case where a sidecar is available before its corresponding Function Container, we want to reach that Container as soon as possible (and we aren't expecting a situation where we ultimately have to give up). Also, the Sidecar has nothing else to do until that Container arrives, so it might as well be aggressive in the retry loop.